packages/freesewing.shared/components/workbench/draft/part/index.js

```diff
@@ -6,7 +6,9 @@ import { getProps } from '../utils'
 const XrayPart = props => {
   // Don't bother if this is the only part on display
   if (props.gist.only && props.gist.only.length === 1) return null
-  const i = Object.keys(props.gist.xray?.reveal).indexOf(props.partName)%10
+  const i = props.gist.xray?.reveal
+    ? Object.keys(props.gist.xray?.reveal).indexOf(props.partName)%10
```

Comment:
why is there a modulo 10 in the end?

Reply:
We use colors here to identify various points on the screen.
A theme defines 10 colors for this:
https://github.com/freesewing/freesewing/blob/d9cceaee72b476d8fde82de797ddfc65079b72ad/packages/freesewing.shared/themes/light.js#L197-L216

So that's why from the 11th onwards, we restart at color 1 (or more correctly, from the 10th, we restart at color 0)